tdms/tests/unit/matlab_benchmark_scripts/benchmark_test_field_interpolation_E.m

```diff
@@ -0,0 +1,164 @@
+%{
+     BENCHMARK FOR E-FIELD INTERPOLATION METHODS
+
+Tests the interpolation accuracy for the E-field components, when
+interpolated in each direction to the centre of a Yee cell - benchmarking
+test_field_interpolation::TEST_CASE("E-field interpolation check").
+
+The E-field used will have identical field components, each of which are
+given by the expression:
+
+E{x,y,z}(xx,yy,zz) = sin(2\pi zz) * exp(-yy^2) * ( 1./ (10xx^2+1) ),
```

Comment:
It would be more realistic if the function was something more like:

E{x,y,z}(xx,yy,zz) = sin(2\pi xx) * exp(-xx^2) *sin(2\pi yy) * exp(-yy^2)* sin(2\pi zz) * exp(-zz^2)

Reply:
Thanks - I'll update the tests to use the more realistic suggestions.
Just to confirm, that's the field
$$E_x(x,y,z) = \sin(2\pi x)e^{-x^2}, \quad E_y(x,y,z) = \sin(2\pi y)e^{-y^2}, \quad E_z(x,y,z) = \sin(2\pi z)e^{-z^2},$$
IE each component is sine multiplied by a Gaussian? (Rather than each component being the product of the three expressions above).

Reply:
Actually, it's more realistic of Ex varied as a function of y and/or z, and similarly for the other components. Just because of the transverse nature of electromagnetic waves.

Reply:
Something cyclic like
$$E_x(x,y,z) = \sin(2\pi y)e^{-y^2}, \quad E_y(x,y,z) = \sin(2\pi z)e^{-z^2}, \quad E_z(x,y,z) = \sin(2\pi x)e^{-x^2},$$
for example?

Reply:
Yes, that's better